### DISCUSSION TITLE
Crash Analyzer

### DISCUSSION THREAD

**Discussion by bingling-sama** [GlobeMC/crashmc.com#127] [开发提案]
关于 Crash Analyzer 后期重做的设计方案

**Comment by bingling-sama** [comment]
目前的 Crash Analyzer 的分析算法极其~~丑陋~~简陋，我们急需一个重做方案。

我目前的构思是将 CA 重构成模块化的，把各启动器的日志适配分成若干个模块，某些重复的（比如 Minecraft 本身的日志分析）算法抽象成 API，用模块将其组织为分析流程。

**Reply by zyxkad** [reply]
我可以提供minecraft的错误及崩溃分析API, 但是JVM级别的崩溃无能为力（
